src/screens/InspectionCapture/index.js

```diff
@@ -27,16 +30,22 @@ export default function InspectionCapture() {
   const { errorHandler } = useMonitoring();
 
   const { inspectionId, taskName, vehicleType, selectedMode } = route.params;
-  const { sights } = useClient();
+  const { Clients, sights } = useClient();
```

Comment:
It should be `clients` instead of 'Clients`.

Reply:
Done